nss-day-cohort-52/giffygram-retro-blasters
# Direct Messages Working/ Favorite Page Added/ Logout Working/ Post Order Fixed / Navigation Fixes / CSS

#### Changes Made
1. Added file `filename` to `directoryname` directory.
1. Modified file `PostList` to include `Check for state for loading favorites page / users` and functionality.
2. Modified file `DirectMessage` to include `Send Messages` and functionality.
3. Modified file `navBar` to include `Logout` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Click Show Favorites should show all your favorites. When unfavorite a post it should disappear from favorites page.
    > Click send message fill out form and messages should appear
    > Clicking on a user now shows messages between you and that user
    > Clicking logout now logs you out
    > When posting messages or posts the newest post should display at the top


4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
